packages/react-text/src/components/Text/Text.test.tsx

```diff
@@ -6,4 +10,160 @@ describe('Text', () => {
     Component: Text,
     displayName: 'Text',
   });
+
+  it('renders a default state', () => {
```

Comment:
Do you plan to add some VR tests in this PR (for example `apps/vr-tests/src/stories/Accordion.stories.tsx`)? IMO that it's a good opportunity to use them and replace these assertions.

Is there a specific need to unit test all permutations?

Reply:
Yes, the plan is to add VR tests to everything but first we needed the implementation done. Since we used TDD, we needed tests to drive the implementation.
As for the specific need, these tests ensure the behavior of the component and VR tests shouldn't be the ones covering that.

Reply:
lets make PRs small as possible, meaning VR tests and related stuff should/could be added in separate PR. thx! 🙏

Reply:
Also part of the reason 💪

Reply:
To be clarify: my comment was mainly about testing applied styles. In Northstar we are relying only Screener to test styles i.e. we don't test styles components in unit tests at all ¯\_(ツ)_/¯ That's why I mentioned VR tests, for my POV it's too _verbose_ for unit tests.

I haven't seen such practice (to test applied styles) in other UI libraries. For sure, you can keep these assertions, but if you think that this approach should be applied to other components I suggest to start a discussion via RFC 🙄 Is there sense in it?

Reply:
The main thing to focus here is that unit tests are for testing the functionality/behavior and styles are just the result of the Text component's functionality. As for VR tests, their purpose is to guarantee the visual accuracy.
I'll follow up on this and create an RFC to clarify what our tests should/shouldn't cover.